modal/README.md

```diff
@@ -0,0 +1,60 @@
+# Modal
+
+## Overview
+
+[Modal][1] lets you run generative AI models, large-scale batch jobs, job
+queues, and much more. This integration collects metrics (CPU, memory
+or GPU use for example) and logs (stdout/stderr logging, audit logs) from your
+applications for you to observe directly in your Datadog account.
```

Comment:
The overview should include: 

1. What does your product do (in 1-2 sentences)? This has been done. 
2. What will customers get from this integration, and why is it valuable to them? This should be clear to new customers who are looking at the tile. 
3. What types of data will your integration send in or pull out, **and what's the value of that data?**

Reply:
Appended a few sentences and re-edited the overview text. It's very general sounding given "modal applications" are effectively any python program, so for instance if a program logged "<N> tasks completed" and then set up a datadog filter for N and graphed that overtime, that's valuable to the author of the application. It's specific to Modal in that we're running the code for you, and emitting those logs to your datadog account, but it's valuable in anyway you choose to use it.